sdkconfig.defaults

```diff
@@ -21,7 +21,7 @@ CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240=y
 # Can't use SRAM1 for IRAM as it conflict with un-updatable bootloaders in the wild :@ .
 # CONFIG_ESP_SYSTEM_ESP32_SRAM1_REGION_AS_IRAM=y
 CONFIG_ESP_CONSOLE_UART_CUSTOM=y
-CONFIG_ESP_CONSOLE_UART_BAUDRATE=921600
+CONFIG_ESP_CONSOLE_UART_BAUDRATE=1000000
```

Comment:
The default baud rate of 921600 does not play nice with Arduino Pro Micro.  The data comes across all garbled.  Tested with the following baud rates and had no problems connecting Arduino.
115200
250000
500000
1000000